utilities/blob_db/blob_db_impl.cc

```diff
@@ -202,6 +202,7 @@ Status BlobDBImpl::Open(std::vector<ColumnFamilyHandle*>* handles) {
   }
 
   ROCKS_LOG_INFO(db_options_.info_log, "BlobDB pointer %p", this);
+  bdb_options_.Dump(db_options_.info_log.get());
```

Comment:
A better idea is to use db_impl_->GetOptions()->info_log.get()

Reply:
GetOptions() come with the overhead of composing ImmutableDBOptions, MutableDBOptions, ImmutableCFOptions, MutableCFOptions into Options, I prefer to avoid calling it.